datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -71,34 +91,122 @@ Where:
+```
+
+This puts the password into a shell variable which is then passed into the
+docker client command with the -e flag which in turn relays the password to the
+DTR bootstrapper.
+
+## Testing backups
+
 To validate that the backup was correctly performed, you can print the contents
 of the tar file created:
 
 ```none
 $ tar -tf /tmp/backup.tar
 ```
 
+The structure of the archive should look something like this:
+
+```none
+dtr-backup-v2.2.1/
+dtr-backup-v2.2.1/rethink/
+dtr-backup-v2.2.1/rethink/events/
+dtr-backup-v2.2.1/rethink/events/0
+dtr-backup-v2.2.1/rethink/events/1
+dtr-backup-v2.2.1/rethink/events/2
+dtr-backup-v2.2.1/rethink/user_settings/
+dtr-backup-v2.2.1/rethink/webhooks/
+dtr-backup-v2.2.1/rethink/properties/
+dtr-backup-v2.2.1/rethink/properties/0
+...
+dtr-backup-v2.2.1/rethink/properties/50
+dtr-backup-v2.2.1/rethink/properties/51
+dtr-backup-v2.2.1/rethink/properties/52
+dtr-backup-v2.2.1/rethink/repositories/
+dtr-backup-v2.2.1/rethink/repositories/0
+dtr-backup-v2.2.1/rethink/blobs/
+dtr-backup-v2.2.1/rethink/blobs/0
+dtr-backup-v2.2.1/rethink/blobs/1
+dtr-backup-v2.2.1/rethink/blobs/2
+dtr-backup-v2.2.1/rethink/blobs/3
+dtr-backup-v2.2.1/rethink/blobs/4
+dtr-backup-v2.2.1/rethink/blobs/5
+dtr-backup-v2.2.1/rethink/blobs/6
+dtr-backup-v2.2.1/rethink/manifests/
+dtr-backup-v2.2.1/rethink/manifests/0
+dtr-backup-v2.2.1/rethink/namespace_team_access/
+dtr-backup-v2.2.1/rethink/repository_team_access/
+dtr-backup-v2.2.1/rethink/tuf_files/
+dtr-backup-v2.2.1/rethink/layers/
+dtr-backup-v2.2.1/rethink/tags/
+dtr-backup-v2.2.1/rethink/tags/0
+dtr-backup-v2.2.1/rethink/private_keys/
+dtr-backup-v2.2.1/rethink/blob_repository/
+dtr-backup-v2.2.1/rethink/blob_repository/0
+dtr-backup-v2.2.1/rethink/blob_repository/1
+dtr-backup-v2.2.1/rethink/blob_repository/2
+dtr-backup-v2.2.1/rethink/blob_repository/3
+dtr-backup-v2.2.1/rethink/blob_repository/4
+dtr-backup-v2.2.1/rethink/blob_repository/5
+dtr-backup-v2.2.1/rethink/blob_repository/6
+dtr-backup-v2.2.1/rethink/content_caches/
+dtr-backup-v2.2.1/rethink/client_tokens/
+dtr-backup-v2.2.1/rethink/client_tokens/0
+```
+
+To really test that the backup works, you must make a copy of your UCP cluster
+by backing it up and restoring it onto separate machines. Then you can restore
+DTR there from your backup and verify that it has all the data you expect to
+see.
+
 ## Restore DTR data
 
-You can restore a DTR node from a backup using the `restore`
-command.
-This command performs a fresh installation of DTR, and reconfigures it with
-the configuration created during a backup.
+You can restore a DTR node from a backup using the `restore` command.
 
-The command starts by installing DTR, restores the configurations stored on
-etcd, and then restores the repository metadata stored on RethinkDB. You
-can use the `--config-only` option, to only restore the configurations stored
-on etcd.
+Note that backups are tied to specific DTR versions and work
```

Comment:
"it works best" is technically true but I think this is one of those situation where we need to take a strong stance and say "it only works with the same version".
I bet we don't have tests to exercise this across versions so we don't provide any guarantees that it will work.